### ISSUE TITLE
How to insert a menu one from the end.

### ISSUE THREAD

**Issue by stuaxo** [linuxmint/xed#306]
Using python and UI_Manager I can easily insert a menu at the end, but how can I insert my menu so it appears before the "Help" menu ?

Making negative indexing for the insertion point insert from the end would be handy, though I realise this is probably more an upstream Gtk thing, unfortunately the UI Manager menu API is deprecated there.

**Comment by collinss** [comment]
There's a placeholder between tools and documents you could use through the UIManager. Unfortunately, if you want to add menu items in the current version of xed, the UIManager is really the only good way to do it, deprecations not withstanding.

**Comment by stuaxo** [comment]
I've come to terms with using the UIManager, since my plugin had to work with various versions of Gedit in the past I've ported that code.

Originally I was going to write a fake GMenu that used UIManager underneath but that was too tricky.

What I have started is making some shims to hide enough of the other differences between Pluma/Xed and Gedit to make my plugin work.

(This is a bit hacky):
https://github.com/shoebot/plugin-shoebot/blob/feature/editors-updated/plugin_shoebot/plugins/peas_editor_shims.py

Cheers, will close this for now.
